### ISSUE TITLE
associated_coordinates() example errors due to missing CRS

### ISSUE THREAD

**Issue by wjones127** [ropensci/unifir#8]
Here's the reprex I ran:

```r
library(unifir)
simulated_data <- data.frame(
  id = seq(1, 100, 1),
  lat = runif(100, 44.04905, 44.17609),
  lng = runif(100, -74.01188, -73.83493)
)
simulated_data <- sf::st_as_sf(simulated_data, coords = c("lng", "lat"))
output_files <- terrainr::get_tiles(simulated_data)
#> Warning in get_tiles.sf(simulated_data): Assuming geographic CRS. Set
#> 'projected' to TRUE if projected.
#> Warning in handle_bboxSR(data, projected): Assuming CRS of EPSG 4326 (set bboxSR
#> explicity to override)
temptiff <- tempfile(fileext = ".tif")
terrainr::merge_rasters(output_files["elevation"][[1]], temptiff)
associate_coordinates(simulated_data, temptiff)
#> Error in st_transform.sfc(st_geometry(x), crs, ...): cannot transform sfc object with missing crs
```

**Comment by mikemahoney218** [comment]
Addressed this in 1d5b1fe -- in addition to explicitly setting the CRS in this example so that it's now correct, I also added CI to run the examples so that failing examples wrapped in `dontrun` will still get caught. Thanks for flagging!

**Comment by wjones127** [comment]
👍  That works for me now with the latest changes.
